src/main.cc

```diff
@@ -15,17 +15,23 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+#include <exception>
+#include <stdexcept>
 #include <cstdio>
 #include <cstdlib>
+#include <fmt/color.h>
 #include <fmt/core.h>
 #include <gflags/gflags.h>
 #include <unistd.h>
+
 #include "io/io.h"
 #include "slide/slide.h"
 
-DEFINE_string(input, "/dev/stdin", "Input markdown file.");
-DEFINE_string(output, "/dev/stdout", "Output directory. Where to place render slide");
+DEFINE_string(input, "", "Inpput markdown file.");
+DEFINE_string(output, "", "Output directory. Where to place rendered slide");
```

Comment:
you can't read from "", can you?
```suggestion
DEFINE_string(input, "./", "Input markdown file.");
DEFINE_string(output, "./", "Output directory. Where to place rendered slide");
```